locales/it_IT/pcsx2_Iconized.po

```diff
@@ -717,22 +715,24 @@ msgstr ""
 "non corrette. Se hai problemi di emulazione, per prima cosa disattiva le "
 "opzioni in questo pannello."
 
-#: pcsx2/gui/Panels/SpeedhacksPanel.cpp:126
-#, fuzzy
+#: pcsx2/gui/Panels/SpeedhacksPanel.cpp:135
 msgid ""
-"Setting higher values on this slider effectively reduces the clock speed of "
-"the EmotionEngine's R5900 core cpu, and typically brings big speedups to "
+"Setting lower values on this slider effectively reduces the clock speed of "
+"the EmotionEngine's R5900 CPU core, and typically brings big speedups to "
 "games that fail to utilize the full potential of the real PS2 hardware. "
-"Conversely, lower values effectively increase the clock speed which may "
+"Conversely, higher values effectively increase the clock speed which may "
 "bring about an increase in in-game FPS while also making games more "
 "demanding and possibly causing glitches."
 msgstr ""
-"Impostando i valori più elevati di questo slider di fatto si riduce la "
-"frequenza della CPU core R5900 dell'EmotionEngine portando a grossi aumenti "
-"di velocità in quei giochi che non riescono ad utilizzare il pieno "
-"potenziale dell'hardware della PS2."
+"Impostando questo slider su valori più bassi viene ridotta la frequenza del "
+"core CPU R5900 dell'EmotionEngine. In questo modo si può riscontrare un "
+"aumento della velocità di emulazione nei giochi che non usavano fino in "
+"fondo l'hardware originale della PS2. Viceversa, valori più elevati vanno ad "
```

Comment:
Sorry to piss again :S
Don't you think *Viceversa* etc, could look better in a new line?